### PR TITLE
Allow GHC-8.4.1 (and other newer deps)

### PR DESCRIPTION
**Pull request by phadej**

### CODE REVIEW

File: cryptohash-sha512.cabal [hunk 1]
```diff
@@ -63,9 +66,9 @@ test-suite test-sha512
 
                    , base16-bytestring >= 0.1.1  && < 0.2
                    , SHA               >= 1.6.4  && < 1.7
-                   , tasty             == 0.11.*
-                   , tasty-quickcheck  == 0.8.*
-                   , tasty-hunit       == 0.9.*
+                   , tasty             >= 0.11   && < 1.1
```
Comment (hvr) [inline cryptohash-sha512.cabal]: As I comented in the somewhat redundant #2 already:

> strictly speaking, this ought to be `== 0.11.* || == 0.12.* || == 1.0.*` assuming you can confirm that it works correctly with all 3 major versions.

Comment (phadej) [reply cryptohash-sha512.cabal]: Made that way, test run successfully with all three majors - tried locally.

**Comment by phadej** [comment]
ping @hvr, I amended the PR

**Comment by hvr** [comment]
thx!

**Pull request merged**